pyproject.toml

```diff
@@ -23,6 +23,7 @@ isort = "^5.10.1"
 flake8 = "^4.0.1"
 pytest = "^6.2.5"
 pytest-cov = "^3.0.0"
+pyyaml = "^6.0"
```

Comment:
if this is for reading in the config, wouldn't this be a dependency, not a dev-dependency?

Reply:
Done in https://github.com/mbhall88/tbpore/pull/8/commits/9ed4fd61dab3335ba4df6a86f95402a9a59f8651